uis/src/com/biglybt/plugin/startstoprules/defaultplugin/ui/swt/ConfigSectionDownloading.java

```diff
@@ -95,12 +95,11 @@ public Composite configSectionCreate(Composite parent) {
 	linkLabel.addMouseListener(new MouseAdapter() {
```

Comment:
Isn't this the same as default behavour in LinkLabel?

Reply:
I see the default behaviour is slightly different - it handles Strings
```
	    	new MouseAdapter() {
		      @Override
		      public void mouseDoubleClick(MouseEvent arg0) {
		    	  launch( arg0 );
		      }
		      @Override
		      public void mouseUp(MouseEvent arg0) {
		    	  launch( arg0 );
		      }
		      private void
		      launch(
		    	  MouseEvent	arg0 )
		      {
		    	  String url = (String) ((Label) arg0.widget).getData();
		    	  
		    	  if ( url != null ){
		    		  Utils.launch( url );
		    	  }
		    	  
		      }
		    }
```

Would be nice if the default implementation would be able to deal with URL's or Wiki enums somehow